Source/WebCore/css/parser/CSSSelectorParser.cpp

```diff
@@ -663,8 +663,8 @@ std::unique_ptr<CSSParserSelector> CSSSelectorParser::consumeNesting(CSSParserTo
     range.consume();
 
     auto selector = makeUnique<CSSParserSelector>();
-    selector->setMatch(CSSSelector::Match::PseudoClass);
-    selector->setPseudoClassType(CSSSelector::PseudoClassNestingParent);
+    selector->setMatch(CSSSelector::Match::NestingParent);
+    selector->setPseudoClassType(CSSSelector::PseudoClassUnknown);
```

Comment:
Do you need this line? It seems to be initialized by default to PseudoClassUnknown: https://searchfox.org/wubkat/rev/d5918187d18a309512bf04da9a709f43762634a0/Source/WebCore/css/CSSSelector.h#328

Reply:
Indeed it's redundant.